documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -4,46 +4,96 @@
+// * xstreams-support
+  * xref:how-debezium-oracle-connectors-handle-faults-and-problems[]
+endif::product[]
+
+// Type: assembly
+// ModuleID: how-debezium-oracle-connectors-work
+// Title: How {prodname} Oracle connectors work
 [[how-the-oracle-connector-works]]
-== How the Oracle Connector Works
+== How the Oracle connector works
 
 To optimally configure and run a {prodname} Oracle connector, it is helpful to understand how the connector performs snapshots, streams change events, determines Kafka topic names, and uses metadata.
 
+ifdef::product[]
+Details are in the following topics:
+
+* xref:how-debezium-oracle-connectors-perform-database-snapshots[]
+* xref:default-names-of-kafka-topics-that-receive-oracle-change-event-records[]
+* xref:about-the-debezium-oracle-connector-schema-change-topic[]
+* xref:debezium-oracle-connector-generated-events-that-represent-transaction-boundaries[]
+
+endif::product[]
+
+// Type: concept
+// Title: How {prodname} Oracle connectors perform database snapshots
+// ModuleID: how-debezium-oracle-connectors-perform-database-snapshots
 [[oracle-snapshots]]
 === Snapshots
 
-Most Oracle servers are configured to not retain the complete history of the database in the redo logs,
-so the {prodname} Oracle connector would be unable to see the entire history of the database by simply reading the logs.
-Consequently, the first time the connector starts, it performs an initial _consistent snapshot_ of the database.
-the default behavior for performing a snapshot consists of the following steps.
-you can change this behavior by setting the `snapshot.mode` connector configuration property to a value other than `initial`.
+Typically, the redo logs on an Oracle server are configured to not retain the complete history of the database.
+As a result, the {prodname} Oracle connector cannot obtain the entire history of the database from the logs.
+To establish a baseline for the current state of the database, the first time that the connector starts, it performs an initial _consistent snapshot_ of the database.
+
+You can configure how the connector creates snapshots by setting the value of the `snapshot.mode` connector configuration property.
+By default, the connector's snapshot mode is set to `initial`.
 
-1. Determine the tables to be captured
-2. Obtain a `ROW SHARE MODE` lock on each of the monitored tables to ensure that no structural changes can occur to any of the tables.
-3. Read the current SCN ("system change number") position in the server's redo log.
+[[default-workflow-for-performing-an-initial-snapshot]]
+.Default workflow for performing an initial snapshot
+Under the default behavior, the connector completes the following tasks to create a snapshot:
+
+1. Determines the tables to be captured
+2. Obtain an `IN EXCLUSIVE MODE` lock on each of the monitored tables to prevent structural changes from occurring during creation of the snapshot. {prodname} holds the locks for only a short time.
```

Comment:
Why did we revert from a `ROW SHARE MODE` lock to `IN EXCLUSIVE MODE` lock after edits?

Reply:
Arrrgh! Sorry, my bad. I missed that when I was pulling in changes from the edits I saved from my earlier attempt at this. Thanks for catching that.